test/paper-dropdown-menu-light.html

```diff
@@ -40,7 +38,7 @@
   <test-fixture id="PreselectedDropdownMenu">
     <template>
       <paper-dropdown-menu-light no-animations>
-        <paper-listbox class="dropdown-content" selected="1">
+        <paper-listbox slot="dropdown-content" class="dropdown-content" selected="1">
```

Comment:
Do you need the `class` in this example? We control the demo, so we know `paper-listbox` will be the same version as `paper-dropdown-menu`